src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -8,6 +8,14 @@ dataverse.db.host=localhost
 dataverse.db.port=5432
 dataverse.db.user=dataverse
 dataverse.db.name=dataverse
+
+# RSERVE
+dataverse.rserve.host=localhost
+dataverse.rserve.port=6311
+dataverse.rserve.username=rserve
+dataverse.rserve.password=rserve
+dataverse.rserve.tempdir=/tmp/Rserve
```

Comment:
`/tmp/Rserv` (without the second `e`) would be a better default - this is the default temp directory for Rserve (and that's what our guide says it defaults too; even though it appears that it was not true!)

Reply:
Done.